File: arrays.js
/* 
    Arrays
*/

let nombres = [];


console.log(typeof(nombres))

nombres.push("Hugo");
nombres.push("Paco");

//nombres.pop()

nombres = nombres.filter((elem) => elem !== 'Hugo')

nombres = nombres.map((elem) => elem.toUpperCase())

console.log(nombres);

File: arrays.py
""" 
    Arrays
"""

nombres = []

print(dir(nombres))

nombres.append("Hugo")
nombres.append("Paco")

#nombres.pop()

nombres = list(filter(lambda elem: elem != 'Hugo', nombres))

nombres = list(map(lambda elem: elem.upper(), nombres))

print(nombres)